packages/services/f-braze-adapter/src/services/BrazeConsumer.js

```diff
@@ -108,7 +110,8 @@ class BrazeConsumer {
             sortByCardOrder,
             cards => filterByEnabledCardTypes(cards, this.enabledCardTypes),
             cards => filterByBrands(cards, this.brands),
-            filterByCurrentlyActive
+            filterByCurrentlyActive,
+            cards => locationFilter(cards, currentLocation)
```

Comment:
We don't want 'For You' cards filtered out when they have a location set, but when there is no currently searched for location. This can be revised later on (i.e. location dropdown solution) but for now please allow this filter to be added for homepage (and therefore HCC) only.

Reply:
I have now added in a check for the URL as discussed